musonant/questioner
# Replace all model methods

#### What does this PR do?
- Replace method in models to use database
#### Description of Task to be completed?
- Remove methods using data structures
- Write methods using the database
#### How should this be manually tested?
- Clone repository and install dependencies by running `npm install`
- Create database `questioner` and make sure it is linked to in the `.env` file
- run `npm run migrate`: This creates new tables in database
- Using postman. test previously included endpoints
- Data returned should be fetched from the database
#### Any background context you want to provide?
NA
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/163250873
#### Screenshots (if appropriate)
#### Questions:`
